base/client.jl

```diff
@@ -30,6 +30,8 @@ answer_color() = text_colors[repl_color("JULIA_ANSWER_COLOR", default_color_answ
 stackframe_lineinfo_color() = repl_color("JULIA_STACKFRAME_LINEINFO_COLOR", :bold)
 stackframe_function_color() = repl_color("JULIA_STACKFRAME_FUNCTION_COLOR", :bold)
 
+alt_light_black_color() = repl_color("JULIA_ALT_LIGHT_BLACK_COLOR", :light_black)
```

Comment:
I'm not sure I like the "alt_light_black" name. Can we come up with a better name?

Reply:
I definitely do not mind any other name, consider it as a placeholder.

Initially, I went with `:stackframe_color` but then decided that `:alt_light_black` is better.